jina/peapods/runtimes/request_handlers/data_request_handler.py

```diff
@@ -112,6 +119,11 @@ def handle(self, requests: List['DataRequest']) -> DataRequest:
         :param requests: The messages to handle containing a DataRequest
         :returns: the processed message
         """
+        docs = get_docs_from_request(
```

Comment:
I think there is no need to move this code up, lets leave it where it was before? at line 133?

Reply:
changed